python/ray/node.py

```diff
@@ -244,7 +244,8 @@ def get_resource_spec(self):
                 self._ray_params.num_cpus, self._ray_params.num_gpus,
                 self._ray_params.memory, self._ray_params.object_store_memory,
                 self._ray_params.resources,
-                self._ray_params.redis_max_memory).resolve(is_head=self.head)
+                self._ray_params.redis_max_memory).resolve(
+                    is_head=self.head, node_ip_address=self.node_ip_address)
```

Comment:
Provided `node_ip_address ` is needed because the `node` instance has not been created right now. `ray.services.get_node_ip_address()` can't get the correct ip address.

Reply:
The [resolve](https://github.com/ray-project/ray/blob/98827c8f4f6b17291bf0f65457f1517ab3bcd8d9/python/ray/tune/ray_trial_executor.py#L487) without provided node_ip_address should be OK because the ray has initialized.